pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+Pull Request Template
+=======
+
+### Step-by-step guide
+How to add a pull request template to your GitHub repository
+
+1. Create a `.github` folder in the root of your repository if it doesn’t exist.
+2. Inside the `.github` folder, create a markdown file named `pull_request_template.md`.
+3. Add the markdown from below to the `pull_request_template.md`.
+4. Commit and push your changes.
+5. Now, whenever someone creates a pull request in your repository, the content of the `pull_request_template.md` file will automatically appear in the pull request description box.
+
+Remember, you can customize the template according to the specific needs of your project. The template below is just a starting point.
+
+***
+
+```
+# Pull Request Template
+
+## Description
+*Add a technical summary of the change, specifying which issue is resolved. Also, include the motivation and context behind the change if relevant.*
+
+## Related Tickets & Documents
+*Provide links to any related tickets or documents.*
+
+## QA Instructions, Screenshots, Recordings
+*Include the steps that need to be followed to test your changes. If applicable, include screenshots or recordings to help explain your PR.*
+
+## Checklist before requesting a review
+*Please leave a comment to the reviewer if something is not applicable.*
+I have:
+- [ ] Reviewed my own code
```

Comment:
Only relevant files were touched

Reply:
@zeger-pon comments have been resolved, all good points of things we have seen happening in the past. So added them all.